code/modules/atmospherics/machinery/components/unary_devices/cryo.dm

```diff
@@ -59,7 +59,7 @@
 		occupant.adjustOxyLoss(-1)
 	if (occupant.getToxLoss())
 		occupant.adjustToxLoss(-1)
-	occupant.heal_limb_damage(1, 1, updating_health = TRUE)
+	occupant.heal_overall_damage(1, 1, updating_health = TRUE) // changed to use heal_overall_damage as cyropods wash over the entire body, not one limb at a time
```

Comment:
Pointless comment but lgtm

Reply:
i'm at the babbling point of learning code. i'll keep my babbling to github comments in the future.

Reply:
Yea get rid of it please